ormar/queryset/utils.py

```diff
@@ -64,7 +64,7 @@ def translate_list_to_dict(  # noqa: CCR001
     for path in list_to_trans:
         current_level = new_dict
         parts = path.split("__")
-        def_val: Any = default
+        def_val: Any = copy.copy(default)
```

Comment:
I would go with deepcopy as it can be nested dict, list of dicts or some mutable custom object

Reply:
Makes sense. Updated and added test